### PR TITLE
Syntax updates based on TS/ES6 guidelines

### PR DESCRIPTION
**Pull request by khofaai**
Apply syntax changing based on :

- https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md
- https://github.com/airbnb/javascript

and separating translation logic and other configs that shouldn't be hard coded

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,14 +2,23 @@ import { NgModule } from '@angular/core';
...
 
 const routes: Routes = [
-  { path: '', redirectTo: 'home', pathMatch: 'full' },
-  { path: 'home', loadChildren: () => import('./home/home.module').then( m => m.HomePageModule)},
+  {
+    path: '',
+    redirectTo: 'home',
+    pathMatch: 'full',
+  },
+  {
```
Comment (akiyamaSM) [inline src/app/app-routing.module.ts]: I still like the inline way

File: src/app/home/home.page.ts [hunk 2]
```diff
@@ -17,32 +17,33 @@ export class HomePage implements OnInit {
...
     let lang = await this.mainService.getLanguage();
     
     lang !== null && (this.langue = lang);
 
-
     let started_at = await this.mainService.getItem('inani_corona_wash_hands_started_at');
-    
-    started_at !== null && (this.started_at = started_at);
+
+    if(started_at != null) {
```
Comment (akiyamaSM) [inline src/app/home/home.page.ts]: Naaaaaaaaaaaaaaaaaaan

**Review by akiyamaSM — CHANGES_REQUESTED** [review]

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (khofaai) [reply src/app/app-routing.module.ts]: the inline way has to much throwback, like you add one `,` and the whole line considered as changing, also readability dude !! inline object like doing this in php : 
```
$obj->key = $value; $obj->keyTwo = $valueTwo; $obj->keyThree = $valueThree;
```

File: src/app/home/home.page.ts (hunk 2 above)
Comment (khofaai) [reply src/app/home/home.page.ts]: you shouldn't do that !!! when you call a function after condition is not like passing values !
duuude readabilityyyy !!

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (akiyamaSM) [reply src/app/app-routing.module.ts:10]: I think its more readble the first one

File: src/app/home/home.page.ts (hunk 2 above)
Comment (akiyamaSM) [reply src/app/home/home.page.ts:38]: Its the same dude ;)

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (khofaai) [reply src/app/app-routing.module.ts:10]: It's one of the most fundamentals for js syntax dude ! it's not about what you think it's about what's right !

File: src/app/home/home.page.ts (hunk 2 above)
Comment (khofaai) [reply src/app/home/home.page.ts:38]: noooooo the first is a `inline condition` the second is a `condition` and `statement` there's a biiig difference

**Pull request closed without merging**